Sources/SimpleSalesforceSDK/SSSDK.swift

```diff
@@ -10,19 +10,19 @@ import SwiftUI
 ///
 /// Once authenticated, you can also use this class to make API calls to Salesforce.
 public class SSSDK {
-
+  
```

Comment:
why do we need this change?

Reply:
Before it was not indented properly so i re-indented using XCode's feature of re-indenting.